server/src/test/java/io/crate/analyze/AlterTableAddColumnAnalyzerTest.java

```diff
@@ -556,4 +556,20 @@ public void test_adding_not_null_column_only_contains_new_column_names_in_constr
 
         assertThat(notNull, Matchers.containsInAnyOrder("dummy"));
     }
+
+    @Test
+    public void test_can_add_fulltext_columns_without_explicit_analyzer() throws Exception {
+        e = SQLExecutor.builder(clusterService)
+            .addTable("CREATE TABLE tbl (x int)")
+            .build();
+        BoundAddColumn addColumn = analyze("ALTER TABLE tbl ADD COLUMN content TEXT INDEX USING FULLTEXT");
+        Map<String, Object> properties = (Map<String, Object>) addColumn.mapping().get("properties");
+        Map<String, Object> content = (Map<String, Object>) properties.get("content");
+
+        assertThat(
+            "Fulltext columns must have type `text`. Regular varchar or text columns have type `keyword`",
+            content.get("type"),
+            is("text")
```

Comment:
Bit confused by docs and tests, need small clarification - in the issue description it states that expected result for dynamic column is `"b" TEXT INDEX USING FULLTEXT;` and fulltext [default](https://crate.io/docs/crate/reference/en/4.6/general/ddl/fulltext-indices.html#fulltext-index-with-analyzer) analyser is [plain/keyword](https://crate.io/docs/crate/reference/en/4.6/general/ddl/analyzers.html#plain). 

From what I have seen in docs I would expect Fulltext default analyser to be is("plain") or  is("keyword") but probably I'm missing something since tests are passing

Reply:
The tests asserts the `type` mapping property, which for text columns can be either `text` (if analyzed) or `keyword` (if not analyzed). It is not checking the analyzer (and `keyword` isn't an analyzer, but a type)

This is indeed a bit confusing because we've the CrateDB side of things but also still the ES side of things.

Reply:
Thanks for the clarification!